rpc-rs/src/rpc_client.rs

```diff
@@ -246,39 +274,33 @@ impl RpcClient {
         let target = target.into();
         let origin_url = origin.url();
         let subject = make_uuid();
-        let issuer = &self.key.public_key();
+        let issuer = self.key.public_key();
         let raw_target_url = target.url();
         let target_url = format!("{}/{}", raw_target_url, &message.method);
 
         // Record all of the fields on the span. To avoid extra allocations, we are only going to
         // record here after we generate/derive the values
-        let current_span = tracing::span::Span::current();
-        current_span.record("issuer", &tracing::field::display(issuer));
-        current_span.record("origin_url", &tracing::field::display(&origin_url));
-        current_span.record("subject", &tracing::field::display(&subject));
-        current_span.record("target_url", &tracing::field::display(&raw_target_url));
-        current_span.record("method", &tracing::field::display(message.method));
-
-        let claims = wascap::prelude::Claims::<wascap::prelude::Invocation>::new(
+        #[cfg(feature = "otel")]
```

Comment:
This would make it so the code below the `let span` line wouldn't compile any more as the cfg block only applies to that line. I don't think this one is needed

Reply:
the code below it uses macros that are disabled if otel is undefined